xps-client/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "xps-client"
+version = "0.1.0"
+edition = "2021"
+authors = ["XMTP Labs <eng@xmtp.com>"]
+description = "XMTP XPS Client for decentralized operations"
+homepage = "https://github.com/xmtp/libxmtp"
+repository = "https://github.com/xmtp/libxmtp"
+license = "MIT"
+keywords = ["ethereum", "did", "did:ethr", "jsonrpc"]
+categories = ["web-programming::websocket", "web-programming::http-server", "web-programming::http-client", "cryptography::cryptocurrencies"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+lib_xps = { git = "https://github.com/xmtp/xps-gateway", branch = "main", package = "lib-xps" }
```

Comment:
Noob question, I don't see `lib-xps` in that repo, what is this package responsible for?

Reply:
yeah it's part of the [reorg PR](https://github.com/xmtp/xps-gateway/pull/57), which splits the main.rs and lib.rs in the current `xps-gateway` into lib-xps and xps

Reply:
jsonrpsee generates a client and server from the defined api 

As it is defined right now, we don't have the client or server feature gated, which is something we have to do before a proper release, so it will pull in both

I'll add the feature gates to the [reorg](https://github.com/xmtp/xps-gateway/pull/57) before it goes in